warehouse/ingest-core/src/main/java/datawave/ingest/util/NGramTokenizationStrategy.java

```diff
@@ -131,48 +127,33 @@ public int tokenize(final NormalizedContentInterface content, int maxNGramLength
                     }
 
                     // Create the tokenizer and tokenizing stream
-                    tokenizer = new NGramTokenizer(2, maxNGramLength);
-                    tokenizer.setReader(reader);
-                    tokenizer.reset();
-                    tokenStream = new ClassicFilter(tokenizer);
-                    tokenStream = new LowerCaseFilter(tokenStream);
-                    tokenStream.addAttribute(CharTermAttribute.class);
-
-                    // Reset the n-gram count
-                    ngramCount = 0;
-
-                    // Increment the tokenizer and applied any generated n-grams
-                    String ngram = null;
-                    while (null != (ngram = this.increment(tokenizer))) {
-                        if (this.updateFilter(ngram, content)) {
-                            ngramCount++;
+                    try (NGramTokenizer tokenizer = new NGramTokenizer(2, maxNGramLength)) {
+                        tokenizer.setReader(reader);
+                        tokenizer.reset();
+                        try (TokenStream tokenStream = new ClassicFilter(tokenizer)) {
```

Comment:
We're only using this tokenStream to form the next one. Thoughts on removing one layer of nesting like this:

```
try (TokenStream tokenStream = new LowerCaseFilter(new ClassicFilter(tokenizer))) {
  // ...
  ```

Reply:
done!